rs/consensus/src/idkg/utils.rs

```diff
@@ -369,51 +369,34 @@ pub(crate) fn inspect_idkg_chain_key_initializations(
             .to_string());
     }
 
-    // TODO(CON-1332): Do not panic if fields are missing
     for ecdsa_init in ecdsa_initializations {
```

Comment:
I think we can also ignore the `ecdsa_initializations` entirely now

Reply:
OK, will remove in a separate PR